styles/main.css

```diff
@@ -21,6 +21,7 @@
 @import "../components/Rating/rating.css";
 @import "../components/Grid/grid.css";
 @import "../components/Toast/toast.css";
+@import "../components/Slider/slider.css";
 
 html {
   font-size: 62.5%;
```

Comment:
Any reason to use percent?

Reply:
To achieve responsiveness. it converts default 16 px size into 10px. so that later he can use font size relative to the body. for ex. now he can use 1 rem instead of 10px.

Reply:
To achieve responsiveness. it converts default 16 px size into 10px. so that later he can use font size relative to the body. for ex. now he can use 1 rem instead of 10px.